configs/config.yaml

```
@@ -1,4 +1,10 @@
 defaults:
   - stac: demo
   - model: rodent
-  - _self_
\ No newline at end of file
+  - _self_
+
+# #FLY_MODEL
+# defaults:
+# - stac: stac_fly_tethered
+# - model: fly_tethered
+# - _self_
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)